### DISCUSSION TITLE
installation blocked by acess not found

### DISCUSSION THREAD

**Discussion by KilaAnge** [brackets-cont/brackets#282] [Q&A]
Hello everyone, please I have been trying to install Brackets on my pc for several times but I have this on the screen:

appdata/roaming/brackets.io/brackets2.0.1/install/brackets1.cab

secure file not found
Please can someone help me ?

**Comment by KilaAnge** [comment]
This is what I get on the screen.

![Capture d’écran (103)](https://github.com/brackets-cont/brackets/assets/136202631/46e6caab-e136-48e2-a2fa-3ec020bbf87f)

**Reply by abose** [reply]
It looks like the installer is corrupted. Please re download the installer from brackets.io and try installation again

**Reply by abose** [reply]
It looks like the installer is corrupted. Please re download the installer from brackets.io and try installation again

**Reply by KilaAnge** [reply]
I did but it still doesn't work and I don't know what to do.
